examples/protocols/multicolor-particle-calibration/artifacts/multicolor-particle-calibration.md

```diff
@@ -24,47 +24,47 @@ Adapted from [https://dx.doi.org/10.17504/protocols.io.bht7j6rn](https://dx.doi.
 2. Provision the stock reagent container containing `Sulforhodamine 101 calibrant`
 3. Provision the stock reagent container containing `Cascade blue calibrant`
 4. Provision the stock reagent container containing `NanoCym 950 nm microspheres`
-5. Transfer 1.0 milliliter of `Phosphate Buffered Saline` sample to wells A1:H12 of  stock reagent container `Fluorescein calibrant`. The reconstituted `Fluorescein` should have a final concentration of 10 uM in `Phosphate Buffered Saline`
+5. Transfer 1.0 milliliter of `Phosphate Buffered Saline` sample to wells A1 of  stock reagent container `Fluorescein calibrant`. The reconstituted `Fluorescein` should have a final concentration of 10 uM in `Phosphate Buffered Saline`
```

Comment:
Awkward to refer to wells of an ordinary lab container (non-plate) that doesn't have wells.